CONTRIBUTING.md

```diff
@@ -231,6 +232,53 @@ To enabled Hot Reloading:
 
 Read more about [Hot Reloading](./docs/local-development.md#hot-reloading)
 
+### Logging
+
+Logging information can be very useful when developing, and there are a few logging options available to you.
+
+To enable logging:
+
+* [Create a local config file](#create-a-local-config-file) if you don't already have one
+* Edit your local config, changing the value of the logger type you want to see to `true`
+
+```json
+  "logging": {
+    "client": false,
+    "firefoxProxy": false,
+    "actions": true
+  }
+```
+
+Let's cover the logging types.
+
+#### client
+
+This option is currently unused.
+
+#### firefoxProxy
+
+This logger outputs a verbose output of all the Firefox protocol packets to your shell. Output looks like this.
+
+```
+WS -> TCP: 99:{"to":"server2.conn4.child5/consoleActor2","type":"startListeners","listeners":["NetworkActivity"]}
```

Comment:
True, there's definitely a balance to strike between brevity and completeness and I wasn't sure about including the examples, figured I'd add them and see how people felt.